### DISCUSSION TITLE
Making Pull Request

### DISCUSSION THREAD

**Discussion by rvragul** [folke/todo-comments.nvim#226] [Q&A]
I want to make a pull request to this repository. I have made some changes in a new branch. Now I want to push my changes.
I executed `git push`. Git tells `git push --set-upstream origin newBranch`. So I ran the command. But I got the following error.
`ERROR: Permission to folke/todo-comments.nvim.git denied to <github user name>.
fatal: Could not read from remote repository.`
What can I do now?

**Comment by cultab** [comment]
This is an old question, but what you tried to do is push changes to this repository, where you don't have permission (only the owner has) you have to do it to your own repository and then make a pull request.
This article from github should be helpful: [link](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request#:~:text=If%20you%20want%20to%20create%20a%20new%20branch%20for%20your%20pull%20request%20and%20do%20not%20have%20write%20permissions%20to%20the%20repository%2C%20you%20can%20fork%20the%20repository%20first.%20For%20more%20information%2C%20see%20%22Creating%20a%20pull%20request%20from%20a%20fork%22%20and%20%22About%20forks.%22)
